code/modules/mob/observer/freelook/marker/signal_abilities/signal_ability.dm

```diff
@@ -162,7 +161,7 @@
 /datum/signal_ability/proc/stop_casting(var/mob/user)
 
 	//Search the user's clickhandlers for any which have an id matching our type, indicating we put them there. And remove those
-	for (var/datum/click_handler/CH in user.GetClickHandlers())
+	for (var/datum/click_handler/CH in user?.GetClickHandlers())
```

Comment:
what does this syntax do? I've never seen it used in this manner

Reply:
It's a left hand null coalescing operator.

if(user && user.GetClickHandlers()) can be simplified to user?.GetClickHandlers()